airflow/providers/amazon/aws/executors/ecs/Dockerfile

```diff
@@ -0,0 +1,86 @@
+# hadolint ignore=DL3007
+FROM apache/airflow:latest
```

Comment:
An example Dockerfile that folks can use with the ECS executor or at least base their images off of to get started.

Reply:
We might consider a different home for this? It's not obvious this is an example to me. Maybe docs, or at least a different path that'll better reflect it?

e.g. https://github.com/apache/airflow/blob/main/docs/apache-airflow/howto/docker-compose/docker-compose.yaml

Reply:
Yupp, eventually creating a robust Docs hierarchy is the goal. But we just wanted to get something minimal in for this alpha release PR. But if you feel very strongly we could consider creating all the docs content in this PR as well. CC @syedahsn

Reply:
We decided to leave this here because we wanted the Dockerfile to exist as a standalone file separate from the documentation. This will make it easier for users to get started, and they can make any changes directly in the file.